layouts/partials/cohorts.html

```diff
@@ -1,6 +1,6 @@
 
 {{ "<!-- details page -->" | safeHTML }}
-<section class="single section-sm pb-0">
+<section class="section single section-sm" style="padding-top: 5vh">
```

Comment:
Curious, did you try this with one of the responsive Bootstrap classes, i.e. `pt-X`? This would be better than hard-coding the style into the layouts. See docs: https://getbootstrap.com/docs/4.6/utilities/spacing/